modules/Bio/EnsEMBL/Mapper.pm

```diff
@@ -341,14 +348,20 @@ sub map_coordinates {
       push( @result, $gap );
       $start = $gap->{'end'} + 1;
     }
-    my ( $target_start, $target_end, $target_ori );
+    my ( $target_start, $target_end);
+    my ( $ori_start, $ori_end);
+    
     my $res;
     if ( exists $pair->{'indel'} ) {
       # When next pair is an IndelPair and not a Coordinate, create the
       # new mapping Coordinate, the IndelCoordinate.
       $target_start = $target_coord->{'start'};
       $target_end   = $target_coord->{'end'};
-
+		
+      #original coordinates
+	  $ori_start = 	$self_coord->{'start'};
```

Comment:
Can I be picky about spacing? We're just starting to get things looking cleaner, with all the past wonky spacing. Thanks.